{"text": "2024-04-30 22:27:48 | INFO    | __main__:stash:131 | run - {'image': '20231121', 'version': '0.10.1.post2', 'role': 'EpicPlayer', 'headless': True}\n", "record": {"elapsed": {"repr": "0:00:01.082549", "seconds": 1.082549}, "exception": null, "extra": {"image": "20231121", "version": "0.10.1.post2", "role": "EpicPlayer", "headless": true}, "file": {"name": "claim.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/claim.py"}, "function": "stash", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 131, "message": "run", "module": "claim", "name": "__main__", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:27:48.218317+08:00", "timestamp": 1714487268.218317}}}
{"text": "2024-04-30 22:27:56 | INFO    | __main__:claim_epic_games:97 | Try to flush cookie - {'task': 'claim_epic_games'}\n", "record": {"elapsed": {"repr": "0:00:08.942371", "seconds": 8.942371}, "exception": null, "extra": {"task": "claim_epic_games"}, "file": {"name": "claim.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/claim.py"}, "function": "claim_epic_games", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 97, "message": "Try to flush cookie", "module": "claim", "name": "__main__", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:27:56.078139+08:00", "timestamp": 1714487276.078139}}}
{"text": "2024-04-30 22:27:58 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:00:11.385625", "seconds": 11.385625}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:27:58.521393+08:00", "timestamp": 1714487278.521393}}}
{"text": "2024-04-30 22:28:00 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xa1 in position 0: invalid start byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic_free_1/epic_free_1/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xa1 in position 0: invalid start byte\n", "record": {"elapsed": {"repr": "0:00:13.097318", "seconds": 13.097318}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xa1 in position 0: invalid start byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xa1 in position 0: invalid start byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:28:00.233086+08:00", "timestamp": 1714487280.233086}}}
{"text": "2024-04-30 22:28:02 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:00:15.318535", "seconds": 15.318535}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:28:02.454303+08:00", "timestamp": 1714487282.454303}}}
{"text": "2024-04-30 22:28:31 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:00:44.862631", "seconds": 44.862631}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:28:31.998399+08:00", "timestamp": 1714487311.998399}}}
{"text": "2024-04-30 22:28:31 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:00:44.863254", "seconds": 44.863254}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:28:31.999022+08:00", "timestamp": 1714487311.999022}}}
{"text": "2024-04-30 22:29:03 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 0}\n", "record": {"elapsed": {"repr": "0:01:15.952981", "seconds": 75.952981}, "exception": null, "extra": {"task": "authorize", "retry": 0}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:29:03.088749+08:00", "timestamp": 1714487343.088749}}}
{"text": "2024-04-30 22:29:05 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:01:18.027176", "seconds": 78.027176}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:29:05.162944+08:00", "timestamp": 1714487345.162944}}}
{"text": "2024-04-30 22:29:07 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xe6 in position 0: invalid continuation byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic_free_1/epic_free_1/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xe6 in position 0: invalid continuation byte\n", "record": {"elapsed": {"repr": "0:01:19.891204", "seconds": 79.891204}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xe6 in position 0: invalid continuation byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xe6 in position 0: invalid continuation byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:29:07.026972+08:00", "timestamp": 1714487347.026972}}}
{"text": "2024-04-30 22:29:09 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:01:22.048709", "seconds": 82.048709}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:29:09.184477+08:00", "timestamp": 1714487349.184477}}}
{"text": "2024-04-30 22:29:38 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:01:51.593078", "seconds": 111.593078}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:29:38.728846+08:00", "timestamp": 1714487378.728846}}}
{"text": "2024-04-30 22:29:38 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:01:51.593632", "seconds": 111.593632}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:29:38.729400+08:00", "timestamp": 1714487378.7294}}}
{"text": "2024-04-30 22:30:09 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 1}\n", "record": {"elapsed": {"repr": "0:02:22.685873", "seconds": 142.685873}, "exception": null, "extra": {"task": "authorize", "retry": 1}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:30:09.821641+08:00", "timestamp": 1714487409.821641}}}
{"text": "2024-04-30 22:30:11 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:02:24.704709", "seconds": 144.704709}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:30:11.840477+08:00", "timestamp": 1714487411.840477}}}
{"text": "2024-04-30 22:30:13 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xda in position 1: invalid continuation byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic_free_1/epic_free_1/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xda in position 1: invalid continuation byte\n", "record": {"elapsed": {"repr": "0:02:26.374340", "seconds": 146.37434}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xda in position 1: invalid continuation byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xda in position 1: invalid continuation byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:30:13.510108+08:00", "timestamp": 1714487413.510108}}}
{"text": "2024-04-30 22:30:15 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:02:28.610814", "seconds": 148.610814}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:30:15.746582+08:00", "timestamp": 1714487415.746582}}}
{"text": "2024-04-30 22:30:45 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:02:58.153992", "seconds": 178.153992}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:30:45.289760+08:00", "timestamp": 1714487445.28976}}}
{"text": "2024-04-30 22:30:45 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:02:58.154595", "seconds": 178.154595}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:30:45.290363+08:00", "timestamp": 1714487445.290363}}}
{"text": "2024-04-30 22:31:16 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 2}\n", "record": {"elapsed": {"repr": "0:03:29.237035", "seconds": 209.237035}, "exception": null, "extra": {"task": "authorize", "retry": 2}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:31:16.372803+08:00", "timestamp": 1714487476.372803}}}
{"text": "2024-04-30 22:31:16 | ERROR    | __main__:run:158 | An error has been caught in function 'run', process 'MainProcess' (3827), thread 'MainThread' (140637278169984): - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic_free_1/epic_free_1/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/home/runner/work/epic_free_1/epic_free_1/epic/src/claim.py\", line 158, in run\n    await agent.stash()\n  File \"/home/runner/work/epic_free_1/epic_free_1/epic/src/claim.py\", line 151, in stash\n    await self.claim_epic_games(context)\n  File \"/home/runner/work/epic_free_1/epic_free_1/epic/src/claim.py\", line 98, in claim_epic_games\n    if await epic.authorize(page):\n  File \"/home/runner/work/epic_free_1/epic_free_1/epic/src/epic_games/agent.py\", line 269, in authorize\n    raise RuntimeError(f\"Failed to flush token - agent={self.__class__.__name__}\")\nRuntimeError: Failed to flush token - agent=EpicGames\n", "record": {"elapsed": {"repr": "0:03:29.365036", "seconds": 209.365036}, "exception": {"type": "RuntimeError", "value": "Failed to flush token - agent=EpicGames", "traceback": true}, "extra": {}, "file": {"name": "claim.py", "path": "/home/runner/work/epic_free_1/epic_free_1/epic/src/claim.py"}, "function": "run", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 158, "message": "An error has been caught in function 'run', process 'MainProcess' (3827), thread 'MainThread' (140637278169984):", "module": "claim", "name": "__main__", "process": {"id": 3827, "name": "MainProcess"}, "thread": {"id": 140637278169984, "name": "MainThread"}, "time": {"repr": "2024-04-30 22:31:16.500804+08:00", "timestamp": 1714487476.500804}}}
